man/systemd.network.xml

```diff
@@ -1643,7 +1643,8 @@
         <varlistentry>
           <term><varname>RequestOptions=</varname></term>
           <listitem>
-            <para>A whitespace-separated list of integers in the range 1–254.</para>
+            <para>When configured, allows to set arbitrary request options in the DHCPv4 request options list and will be
+            sent to the DHCPV4 server. A whitespace-separated list of integers in the range 1–254. Defaults to unset.</para>
```

Comment:
doesn't matter much, but I think in most cases we wrote ranges so far as 1…254 instead of 1-254 (with a pretty unicode `…`), we should do that here too, i guess.